src/authorizations/components/redesigned/CustomApiTokenOverlay.tsx

```diff
@@ -1,7 +1,15 @@
-import React, {FC} from 'react'
+import React, {FC, useState} from 'react'
```

Comment:
This is the component that should render when `Custom Api Token`. I think something happened during rebase.